Source/GTMSessionFetcher.m

```diff
@@ -1689,10 +1698,93 @@ - (void)authorizer:(id<GTMFetcherAuthorizationProtocol>)auth
     @synchronized(self) {
       _request = authorizedRequest;
     }
-    [self beginFetchMayDelay:NO mayAuthorize:NO];
+    [self beginFetchMayDelay:NO mayAuthorize:NO mayDecorate:YES];
   }
 }
 
+- (void)applyDecoratorsAtRequestWillStart:(NSArray<id<GTMFetcherDecoratorProtocol>> *)decorators
+                          startingAtIndex:(NSUInteger)index {
+  GTMSessionCheckNotSynchronized(self);
+  if (index == decorators.count) {
```

Comment:
`>=`, just for caution's sake?

(Same comment in the other `apply…` method)

Reply:
Sure! Done.